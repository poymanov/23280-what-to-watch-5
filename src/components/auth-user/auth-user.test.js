import React from "react";
import renderer from "react-test-renderer";
import AuthUser from "./auth-user";

const currentUser = {
  id: 1,
  name: `test`,
  email: `test`,
  avatarUrl: `test`
};

it(`Should AuthUser render correctly`, () => {
  const tree = renderer
    .create(<AuthUser currentUser={currentUser} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
