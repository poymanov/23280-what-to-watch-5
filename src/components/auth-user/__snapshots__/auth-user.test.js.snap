// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should AuthUser render correctly 1`] = `
<div
  className="user-block"
>
  <div
    className="user-block__avatar"
  >
    <img
      alt="test"
      height="63"
      src="test"
      width="63"
    />
  </div>
</div>
`;
